### PR TITLE
remove extra .gitignores

### PR DESCRIPTION
**Pull request by EvanBacon**
reducing clutter

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -14,4 +16,33 @@ node_modules
 *.tsbuildinfo
 .history
 
+**/*/build
+
+# Logs
+logs
```
Comment (fson) [inline .gitignore:22]: Does this have an effect on https://github.com/expo/expo-cli/tree/f88f72faa5845e12827d24534c8e171be97e92b7/packages/xdl/src/logs ? And does some package actually use this path to store logs?

File: .gitignore [hunk 2]
```diff
@@ -14,4 +16,33 @@ node_modules
...
+*.seed
+
+# Directory for instrumented libs generated by jscoverage/JSCover
+lib-cov
+
+# Coverage directory used by tools like istanbul
 coverage/
+
+# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
+.grunt
```
Comment (fson) [inline .gitignore:37]: I believe this can be removed as we're not using Grunt.

File: .gitignore [hunk 3]
```diff
@@ -14,4 +16,33 @@ node_modules
...
+
+# node-waf configuration
+.lock-wscript
+
+# Compiled binary addons (http://nodejs.org/api/addons.html)
+build/Release
```
Comment (fson) [inline .gitignore:43]: This could already be covered by `packages/*/build/`?

File: .gitignore (hunk 1 above)
Comment (EvanBacon) [reply .gitignore]: it doesn't but I made the path a bit more specific anyways since it seems like it could cause issues in the future.

**Pull request merged**